addons/mail/models/res_users.py

```diff
@@ -316,5 +316,11 @@ def systray_get_activities(self):
                 if activity.state in ("today", "overdue"):
                     user_activities[model_name]["total_count"] += 1
         if "mail.activity" in user_activities:
-            user_activities["mail.activity"]["name"] = _("Other activities")
+            user_activities["mail.activity"].update({
+                "name": _("Other activities"),
+                "views": [
```

Comment:
Have to say this is a bit hardcore... Maybe we could instead define an action_xml_id and have an action somewhere that forces the use of these views instead?

Not really self-speaking to pass on a "views" key that is the direct input expected by the JS "doAction".
Especially since it's ignored unless we are on mail.activity.

Reply:
Yes I hesitated to change it that way (but was lacking a bit of time). I have changed it and it is indeed  better.